src/python/strelka/scanners/scan_lnk.py

```diff
@@ -0,0 +1,471 @@
+import uuid
+import io
+from construct import Struct, Int16ul, GreedyRange, Bytes, StringEncoded, this, Int32ul, If, Enum, CString, IfThenElse, BitsSwapped, BitStruct, Flag, Int32sl, Int8ul
+
+from strelka import strelka
+
+class ScanLNK(strelka.Scanner):
+    #Collects metadata and extracts text from lnk files.
```

Comment:
Recommend changing out the docstring with something like the following:
  
  """Collects metadata from LNK Files."""